thrift-confignode/src/main/thrift/confignode.thrift

```diff
@@ -49,13 +49,26 @@ struct TGlobalConfig {
 }
 
 struct TRatisConfig {
-  1: optional i64 appenderBufferSize
-  2: optional i64 snapshotTriggerThreshold
-  3: optional bool logUnsafeFlushEnable
-  4: optional i64 logSegmentSizeMax
-  5: optional i64 grpcFlowControlWindow
-  6: optional i64 leaderElectionTimeoutMin
-  7: optional i64 leaderElectionTimeoutMax
+  1: optional i64 schemaAppenderBufferSize
+  2: optional i64 dataAppenderBufferSize
+
+  3: optional i64 schemaSnapshotTriggerThreshold
+  4: optional i64 dataSnapshotTriggerThreshold
+
+  5: optional bool schemaLogUnsafeFlushEnable
+  6: optional bool dataLogUnsafeFlushEnable
+
+  7: optional i64 schemaLogSegmentSizeMax
+  8: optional i64 dataLogSegmentSizeMax
+
+  9: optional i64 schemaGrpcFlowControlWindow
+  10: optional i64 dataGrpcFlowControlWindow
+
+  11: optional i64 schemaLeaderElectionTimeoutMin
+  12: optional i64 dataLeaderElectionTimeoutMin
+
+  13: optional i64 schemaLeaderElectionTimeoutMax
+  14: optional i64 dataLeaderElectionTimeoutMax
```

Comment:
required is better? Since the ratis configurations are always assigned by ConfigNode.